src/wad/image.rs

```diff
@@ -77,36 +87,51 @@ impl Image {
 
                 // The second byte is the length of this run. Skip an additional
                 // byte which is ignored for some reason.
-                let run_length =
-                    *try!(source.next()
-                                .to_err_with(|| format!("missing run length: column {}, run {}",
-                                                        i_column, i_run))) as usize;
+                let run_length = *try!(source.next()
+                                             .to_err_with(|| {
+                                                 format!("missing run length: column {}, run {}",
+                                                         i_column,
+                                                         i_run)
+                                             })) as usize;
 
                 // Check that the run fits in the image.
                 if row_start + run_length > height {
-                    return Err(format!("run out of bounds: column {}, run {} ({}, +{}), \
-                                        size {}x{}", i_column, i_run, row_start, run_length,
-                                       width, height).into());
+                    return Err(format!("run out of bounds: column {}, run {} ({}, +{}), size \
+                                        {}x{}",
+                                       i_column,
+                                       i_run,
+                                       row_start,
+                                       run_length,
+                                       width,
+                                       height)
+                                   .into());
                 }
 
                 // An ignored padding byte.
                 try!(source.next()
-                           .to_err_with(|| format!("missing padding byte 1: column {}, run {}",
-                                                   i_column, i_run)));
+                           .to_err_with(|| {
+                               format!("missing padding byte 1: column {}, run {}", i_column, i_run)
+                           }));
 
-                // Iterator to the beginning of the run in `pixels`. Guaranteed to be in bounds by
+                // Iterator to the beginning of the run in `pixels`. Guaranteed to be in bounds
+                // by
                 // the check above.
```

Comment:
Please concatenate with the previous line.

I think if deciding to break a comment, rustfmt should reflow the whole thing unless the next line starts with capital letter or something like that. Any thoughts @nrc ?